platform/src/main/java/org/stellar/anchor/platform/paymentobserver/StellarPaymentObserver.java

```diff
@@ -124,8 +122,8 @@ public static Builder builder() {
 
   public static class Builder {
     String horizonServer = "https://horizon-testnet.stellar.org";
-    List<String> accounts = new LinkedList<>();
-    List<PaymentListener> observers = new LinkedList<>();
+    Set<String> accounts = new TreeSet<>();
```

Comment:
Can you share the reason to change from List to Set? Is it because we don't want to listen to duplicated accounts?

Set is un-sorted and the order can be indeterministic. If we are to use Set, can we use SortedSet?

Reply:
Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/260/commits/9739317e6dbecc16a2d496d42be1e670c6d51b96.